src/nxdoc/nuxeo-server/installation.md

```diff
@@ -41,11 +48,9 @@ Using a Docker image has several advantages:
 - It's what we're using internally at Nuxeo, especially in [Nuxeo Cloud]({{page space='nuxeo-cloud'}}).
 - Docker and the container ecosystem are widely maintained by the open source community.
 
-Before installing the nuxeo Platform, make sure you have a look at the [Hardware and Software Requirements](#hardware-and-software-requirements) in the section below. You may also want to read our page regarding [required storage space]({{page page='how-to-estimate-volume-usage'}}).
+## Nuxeo ZIP Distribution
 
-{{#> callout type='info' heading='Tomcat server ZIP'}}
-Though it is not the recommended way, you can also install the Nuxeo Platform with the [Tomcat server ZIP]({{page page='tomcat-server-zip'}}) archive, which works on any operating system that supports a Java Development Kit.
-{{/callout}}
+You can also install the Nuxeo Platform with the [Tomcat server ZIP]({{page page='tomcat-server-zip'}}) archive, which works on any operating system that supports a Java Development Kit. Before installing the Nuxeo Platform, make sure you have a look at the [Hardware and Software Requirements](#hardware-and-software-requirements) in the section below. You may also want to read our page regarding [required storage space]({{page page='how-to-estimate-volume-usage'}}).
```

Comment:
> Before installing the Nuxeo Platform, make sure you have a look at the [Hardware and Software Requirements](#hardware-and-software-requirements) in the section below. You may also want to read our page regarding [required storage space]({{page page='how-to-estimate-volume-usage'}}).

This doesn't only apply to the ZIP distribution. The requirements specific to the ZIP distribution are in the dedicated page.